openmetadata-service/src/main/java/org/openmetadata/service/Entity.java

```diff
@@ -99,6 +99,8 @@ public final class Entity {
   public static final String TEST_SUITE = "testSuite";
   public static final String TEST_CASE = "testCase";
   public static final String WEB_ANALYTIC_EVENT = "webAnalyticEvent";
+  public static final String DATA_INSIGHT_CHART = "dataInsightChart";
```

Comment:
should we call this metric or chart?

Reply:
I used chart it represents the actual chart. Inside a chart we have metrics and dimensions